TECHNICAL_VALUES_AND_PRIORITIZATION.md

```diff
@@ -0,0 +1,65 @@
+# Technical Values and Priorities
+
+This document outlines the project's technical values and relative
+prioritization used when making tradeoffs.
+
+## Values and priority level
+
+This is only a subset of the overall priorities, listing only
+those which are at the top of the priority list. Anything on this
+list is very important for the project despite the relative priorities shown.
+
+* Priority 1 - Developer experience
+* Priority 2 - Stability
+* Priority 3 - Operational qualities
+* Priority 4 - the Node.js maintainer experience
+* Priority 5 - Technology and API currency
+
+## Value descriptions
+
+### Developer Experience
+We value ensuring that developers are productive and enjoy developing
+with Node.js. Some key elements of this include:
+* Approachability (both technical and community)
+* Great Documentation
+* Bundling friction-reducing APIs and components, even though
+  they could be provided externally
+* Enabling/supporting external packages to ensure overall developer experience
+
+### Stability
+To avoid introducing churn into the Node.js ecosystem, we value stability
+and consistency across releases and avoid breaking changes.
+Some key elements of this include:
+* Backwards compatibility
+* Predictable and stable releases
+* A strong safety net, including explicitly testing how changes
+  in Node.js affect userland packages.
+* Careful consideration of what goes into LTS releases
+
+### Operational Qualities
+We value keeping Node.js safe, performant and lightweight as
+well as the ability to investigate and debug problems in
+development and production. Some key elements of this include:
+* Throughput (speed)
```

Comment:
🤔 Throughput and speed are not the same thing. 

```suggestion
* High throughput
```

Reply:
```suggestion
* High throughput (speed)
```